<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chart3DWindow</class>
 <widget class="QWidget" name="Chart3DWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QSpinBox" name="rowsSpinBox">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Coloring Style</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Rows</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="showOutlineCheckBox">
           <property name="text">
            <string>Show Outline</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="Widgets::ColorPushButton" name="colorButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="coloringStyleCombo">
           <item>
            <property name="text">
             <string>Scalar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="randomButton">
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="3">
          <widget class="QTableView" name="dataTable">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QGroupBox" name="settingsGroupBox">
           <property name="title">
            <string>Settings</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="settingsLayout"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="numOfLabelsSpinBox">
           <property name="minimum">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Number of Labels</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="Widgets::ColorPushButton" name="planesColorButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="showGridLinesCheckBox">
           <property name="text">
            <string>Show Grid Lines</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="showPlanesCheckBox">
           <property name="text">
            <string>Show Planes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="Widgets::ColorPushButton" name="gridLinesColorButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Import File</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Export File</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="importFileEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="exportFileEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="importFileButton">
           <property name="text">
            <string>Open ...</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QPushButton" name="exportFileButton">
           <property name="text">
            <string>Open ..</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="previewLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/colorpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
